### PR TITLE
Removing CRA

### PR DESCRIPTION
**Pull request by aronchick**
Create react app was outdated when we even started. This needs to be ejected.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -54,5 +54,6 @@ coverage.out
 docker/.images
 docker/.pulled
 clients/pyproject.toml
-webui/.yarn/install-state.gz
+install-state.gz
+install-state.gz
```
[the remark this answers is not in the document]
Comment (aronchick) [reply .gitignore:58]: yeah, this was annoying - fixed.

File: .github/workflows/docs.yaml [hunk 2]
```diff
@@ -20,15 +27,22 @@ jobs:
...
+      - name: Install corepack, nodejs, yarn
+        run: |
+          corepack enable
+          asdf reshim nodejs
+          yarn install
         working-directory: docs
       - run: yarn run build
         working-directory: docs
-      - uses: actions/upload-pages-artifact@v2
+      - uses: actions/upload-artifact@v3
```
[the remark this answers is not in the document]
Comment (aronchick) [reply .github/workflows/docs.yaml:45]: great catch - fixed

File: .gitignore [hunk 3]
```diff
@@ -54,5 +54,6 @@ coverage.out
 docker/.images
 docker/.pulled
 clients/pyproject.toml
-webui/.yarn
+install-state.gz
+!webui/config/env.js
 install-state.gz
```
Comment (simonwo) [inline .gitignore:59]: Still two of these...

File: webui/.yarn/yarn.build.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "comment": "This is an auto-generated file, it keeps track of whats been built. This is a local file, don't store this in version control.",
+  "packages": {}
+}
```
Comment (simonwo) [inline webui/.yarn/yarn.build.json:4]: This file claims it should not be checked in

**Review by simonwo — APPROVED** [review]

**Pull request closed without merging**